worksheet_B/PseudoCode.txt

```diff
@@ -0,0 +1,31 @@
+Main Procedure Fallout Game
+    Print "Select how many letter words"
+    Assign user input to variable "CharacterNum"
+    Create variable in "SimilarityScore"
+    Assign SimilarityScore value 0
+    Select random word from array
+    Store random word in variable "RandomWord"
+    
+    Begin check loop{
+    Print list of words to user
+    User input stored as "ChosenWord"
+    if first letter is the same in both RandomWord and Chosen Word 
+        increase SimilarityScore by 25

```

Comment:
Why does a correct letter increase the score by 25 and an incorrect letter decrease it by 20? The worksheet says that the similarity score is "defined as the number of letters which match the secret word", so a correct letter should increase it by 1 and an incorrect letter should not change it.